src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {`${person.isMarried === false ? 'I am not married'
+        : `${person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`}`}`
+      }
+    </p>
```

Comment:
1. you can simplify condition
2. move person status to a variable

```
const personStatus = sex === 'm' ? 'wife' : 'husband' 
```
```suggestion
    <p className="Person__partner">
      {`!isMarried 
        ? 'I am not married'
        : `${partnerName} is my ${personStatus}`
      }
    </p>
```